src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt

```diff
@@ -70,6 +70,7 @@ class RsFileStub : PsiFileStubImpl<RsFile> {
             override fun skipChildProcessingWhenBuildingStubs(parent: ASTNode, child: ASTNode): Boolean {
                 val elementType = child.elementType
                 return elementType == MACRO_ARGUMENT || elementType == MACRO_BODY ||
+                    elementType in RS_DOC_COMMENTS ||
```

Comment:
No, because there are no stubs in comments. We can return `true` from `skipChildProcessingWhenBuildingStubs` for any element type that does not contain stubs and this does not require to bump the stub version.